backend/src/main/java/ch/puzzle/pcts/model/error/GenericError.java

```diff
@@ -0,0 +1,21 @@
+package ch.puzzle.pcts.model.error;
+
+import java.util.Map;
+
+public class GenericError {
```

Comment:
[high​-cmplx]: code is too complex the model does not provide something here as we do not want to use this in the backend for any reason so a dto is enough we do not need an Model for this.